src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -457,6 +457,14 @@ public static boolean doesEventValueHaveTimeStates(Class<? extends Event> event,
 		return getEventValueConverter(event, c, TIME_PAST, false) != null || getEventValueConverter(event, c, TIME_FUTURE, false) != null;
 	}
 
+	/**
+	 * All supported time states for an event value.
+	 * @return An array of all the time states.
+	 */
+	public static int[] getTimeStates() {
```

Comment:
is this being used?

Reply:
yeah in the method below it, havent pushed yet tho :)